source/ranges.tex

```diff
@@ -863,8 +867,9 @@
 
 \begin{itemdescr}
 \pnum
-Since the difference between \libconcept{Range} and \libconcept{View} is largely
-semantic, the two are differentiated with the help of \tcode{enable_view}.
+\tcode{T} models \libconcept{View} only if the complexity
+for each initialization and assignment required by \libconcept{Copyable<T>}
+is constant time.
```

Comment:
This may well be intended, but I don't see how it's editorial. Is this somehow justified by the existing text?

Reply:
Paragraph 1 above: "The `View` concept specifies the requirements of a `Range` type that has constant time copy, move, and assignment operators...".

Although I wouldn't mind sending this to LWG: we need to add the missing requirement that destruction is O(1) as well which is *certainly* not editorial.

Reply:
Let's send the whole thing to LWG then. If they want us to apply the non-normative change editorially we can, but I think it's generally better for holistic changes with a normative component to be applied together.